IntelFsp2WrapperPkg/IntelFsp2WrapperPkg.dsc

```diff
@@ -60,16 +60,14 @@
 ##MSCHANGE Begin
 [LibraryClasses.common]
   BaseBinSecurityLib|MdePkg/Library/BaseBinSecurityLibNull/BaseBinSecurityLibNull.inf
-!if $(TOOL_CHAIN_TAG) == VS2017 or $(TOOL_CHAIN_TAG) == VS2015 or $(TOOL_CHAIN_TAG) == VS2019
-!if $(TARGET) == DEBUG

```

Comment:
Is the removal of TARGET == DEBUG intentional?

Reply:
Same question

Reply:
Yes, stack cookies should be present on release builds as well